detector/aws_route_invalid_vpc_peering_connection.go

```diff
@@ -9,22 +9,20 @@ import (
 
 type AwsRouteInvalidVpcPeeringConnectionDetector struct {
 	*Detector
-	IssueType             string
-	TargetType            string
-	Target                string
-	DeepCheck             bool
 	vpcPeeringConnections map[string]bool
 }
 
 func (d *Detector) CreateAwsRouteInvalidVpcPeeringConnectionDetector() *AwsRouteInvalidVpcPeeringConnectionDetector {
-	return &AwsRouteInvalidVpcPeeringConnectionDetector{
+	nd := &AwsRouteInvalidVpcPeeringConnectionDetector{
 		Detector:              d,
-		IssueType:             issue.ERROR,
-		TargetType:            "resource",
-		Target:                "aws_route",
-		DeepCheck:             true,
 		vpcPeeringConnections: map[string]bool{},
 	}
+	nd.Name = "aws_route_invalid_vpc_peering_connection"
+	nd.IssueType = issue.ERROR
+	nd.TargetType = "resource"
+	nd.Target = "aws_route"
```

Comment:
__[gometalinter]__
* [goconst] 8 other occurrence(s) of "aws_route" found in: aws_route_specified_multiple_targets.go:17:14 aws_route_invalid_nat_gateway.go:23:14 aws_route_invalid_network_interface.go:23:14 aws_route_not_specified_target.go:17:14 aws_route_invalid_instance.go:23:14 aws_route_invalid_egress_only_gateway.go:23:14 aws_route_invalid_route_table.go:23:14 aws_route_invalid_gateway.go:23:14

<!-- Comment by [SideCI](https://sideci.com) -->